src/backend/settings/test.py

```diff
@@ -2,11 +2,11 @@
 
 SECRET_KEY = "CorrectHorseBatteryStaple"
 
-MAIL["SEND"] = False
-
 FRONTEND_URL = "http://example.com/"
 DOMAIN = "example.com"
 
+EMAIL_BACKEND = "anymail.backends.test.EmailBackend"
```

Comment:
test settings should also have email send set to false

Reply:
Resolved in fef64d193369f01dbbdd5790eac8ba3dd1d9c532